lib/widgets/compass/compass_widget.dart

```diff
@@ -0,0 +1,109 @@
+import 'dart:math';
+
+import 'package:chirp_nets/models/user.dart';
+import 'compass_painter.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_compass/flutter_compass.dart';
+
+class CompassWidget extends StatefulWidget {
+  const CompassWidget({
+    this.userLocations,
+    this.currentUser,
+  });
+
+  final List<Map<String, dynamic>> userLocations;
+  final User currentUser;
+
+  @override
+  _CompassWidgetState createState() => _CompassWidgetState();
+}
+
+class _CompassWidgetState extends State<CompassWidget>
+    with SingleTickerProviderStateMixin {
+  final double width = 250;
+  final double height = 250;
```

Comment:
Should these be hardset? how does this scale for different phone sizes? Tablets?